packages/x-date-pickers/src/DesktopDatePicker/DesktopDatePicker.test.tsx

```diff
@@ -68,18 +68,29 @@ describe('<DesktopDatePicker />', () => {
 
   it('allows to change selected date from the input according to `format`', () => {
     const onChangeMock = spy();
-    render(
-      <DesktopDatePicker
-        renderInput={(props) => <TextField placeholder="10/10/2018" {...props} />}
-        label="Masked input"
-        inputFormat="dd/MM/yyyy"
-        value={adapterToUse.date('2018-01-01T00:00:00.000Z')}
-        onChange={onChangeMock}
-        InputAdornmentProps={{
-          disableTypography: true,
-        }}
-      />,
-    );
+    const Test = () => {
```

Comment:
You should be able to use `WrappedDesktopDatePicker` which does exactly what you reimplement here (handled an internal state and expose an `onChange` if you want to track calls)